app/views/events/_future.html.erb

```diff
@@ -0,0 +1,16 @@
+<div class="m-5">
+<% if Event.future.nil? %>
+  <%= "There's not previous events" %>
+  <%= link_to 'Create event', create_event_path %>
+<% else %>
+<% Event.future.each do |inv| %>
```

Comment:
You should make use of a variable instead. Check the comment of the event index controller.